src/components/DisplayNames/DisplayNamesWithTooltip.tsx

```diff
@@ -1,17 +1,17 @@
-import lodashGet from 'lodash/get';
 import React, {Fragment, useCallback, useRef} from 'react';
-import {View} from 'react-native';
-import _ from 'underscore';
+import {Text as RNText, View} from 'react-native';
 import Text from '@components/Text';
 import Tooltip from '@components/Tooltip';
 import styles from '@styles/styles';
-import {defaultProps, propTypes} from './displayNamesPropTypes';
 import DisplayNamesTooltipItem from './DisplayNamesTooltipItem';
+import DisplayNamesProps from './types';
 
-function DisplayNamesWithToolTip(props) {
-    const containerRef = useRef(null);
-    const childRefs = useRef([]);
-    const isEllipsisActive = lodashGet(containerRef.current, 'offsetWidth') < lodashGet(containerRef.current, 'scrollWidth');
+type HTMLElementWithText = HTMLElement & RNText;
+
+function DisplayNamesWithToolTip({shouldUseFullTitle, fullTitle, displayNamesWithTooltips, textStyles = [], numberOfLines = 1}: DisplayNamesProps) {
+    const containerRef = useRef<HTMLElementWithText>(null);
```

Comment:
just wondering if this will be always a html element ?

Reply:
@kubabutkiewicz If I type it with just Text it gives errors, because Text ref doesn't have `getBoundingClientRect`, `offsetWidth`, `scrollWidth` props

![image](https://github.com/Expensify/App/assets/23176449/098fe7ca-0e3b-46c7-ba79-2e82c55ebbb2)